.github/workflows/nightly-normal-session-e2e-tests.yml

```diff
@@ -1,7 +1,11 @@
 ---
-name: E2E synthetic network tests (Nightly pipeline)
+name: Nightly pipeline normal session e2e tests
 
 on:
+  # temp, remove before merge
```

Comment:
Will be removed when https://github.com/Cardinal-Cryptography/aleph-node/actions/runs/5935960293/job/16095914889?pr=1363 finishes

Reply:
It passed: https://github.com/Cardinal-Cryptography/aleph-node/actions/runs/5935960293/job/16095914889?pr=1363